webpack.config.js

```diff
@@ -63,6 +63,9 @@ module.exports = {
     devtoolModuleFilenameTemplate: (info) => path.resolve(info.absoluteResourcePath),
   },
   resolve: {
+    alias: {
+      common: path.resolve(__dirname, "common"),
+    },
```

Comment:
Hm, why is this necessary when we have `modules: [path.resolve(".")` below?

Reply:
I'll remove this